CIME/test_scheduler.py

```diff
@@ -967,7 +967,10 @@ def _xml_phase(self, test):
                 )
             envtest.set_initial_values(case)
             case.set_value("TEST", True)
-            case.set_value("SAVE_TIMING", self._save_timing)
+            if is_perf_test(test):
+                case.set_value("SAVE_TIMING", True)
+            else:
+                case.set_value("SAVE_TIMING", self._save_timing)
```

Comment:
This is the key change I think.  E3SM outputs a bunch of extra perf data and, if you use create_test, none of it is saved.  If you use create_newcase, its saved to a special directory on each machine.  You can override that with a command-line argument but that would do it for every test in a suite. This allows another way to save the timing info from single tests in a large suite (our integration suite has 100+ cases).  Instead of creating/running a separate suite just for performance.